plugins/com.google.cloud.tools.eclipse.appengine.deploy/src-test/com/google/cloud/tools/eclipse/appengine/deploy/AppEngineDeployOutputTest.java

```diff
@@ -25,7 +25,7 @@
 public class AppEngineDeployOutputTest {
 
   @Rule public ExpectedException thrown = ExpectedException.none();
-
+  
```

Comment:
Fails for me: https://travis-ci.org/GoogleCloudPlatform/google-cloud-eclipse/jobs/227602948#L1129